ios/lib/ReactNativeCameraKit/CKCamera.m

```diff
@@ -95,13 +96,21 @@ @interface CKCamera () <AVCaptureFileOutputRecordingDelegate, AVCaptureMetadataO
 // frame for Scanner
 @property (nonatomic, strong) NSDictionary *scannerOptions;
 @property (nonatomic) BOOL showFrame;
-@property (nonatomic) UIView *greenScanner;
+@property (nonatomic) UIView *laserView;
+@property (nonatomic) UIColor *laserViewColor;
+
+@property (nonatomic) CGFloat frameWidth;
+@property (nonatomic) CGFloat frameHeight;
+@property (nonatomic) CGFloat frameLeft;
+@property (nonatomic) CGFloat frameTop;
+@property (nonatomic) UIColor *frameColor;
+
+@property (nonatomic) UIView *overlayView;
+@property (nonatomic) UIColor *overlayColor;
 
-@property (nonatomic) CGFloat frameOffset;
-@property (nonatomic) CGFloat heightFrame;
-@property (nonatomic, strong) UIColor *frameColor;
 @property (nonatomic) UIView *dataReadingFrame;
 @property (nonatomic) UIView *perfomanceBackground;
+@property (nonatomic) UIColor *perfomanceBackgroundColor;
```

Comment:
I don't understand what this property name means

Reply:
changed for surfaceColor